cmd/basin/publication.go

```diff
@@ -190,10 +192,90 @@ func newPublicationStartCommand() *cli.Command {
 	}
 }
 
+func newPublicationUploadCommand() *cli.Command {
```

Comment:
It's a little awkward that this command picks up the provider url from the config. I think it's because the config only works for a single publication (it will get overwritten when you create more than one). Maybe it should handle more than one? ie, be an array of pub configs?

```
dbs:
    postgres:
        user: basin
        password: basin
        host: 127.0.0.1
        port: 5432
        database: mydb
provider_host: 127.0.0.1:3000
```
could instead be something like:

```
publications:
  foo:
    type: postgres
    user: dog_owner
    password: bark
    host: 127.0.0.1
    port: 5432
    database: dogs
    provider_host: 127.0.0.1:3000
  bar:
    type: postgres
    user: cat_owner
    password: meow
    host: 127.0.0.1
    port: 5432
    database: cats
    provider_host: some.other.host.com
```

Reply:
nice catch! for sure, it should be an array of publications